acceptance/tests/facts/debian.rb

```diff
@@ -7,10 +7,23 @@
 # Facts tested: os, processors, networking, identity, kernel
 #
 
-confine :to, :platform => /debian-squeeze|debian-wheezy/
+confine :to, :platform => /debian-squeeze|debian-wheezy|debian-jessie/
 
 agents.each do |agent|
-  os_version = agent['platform'] =~ /squeeze/ ? '6' : '7'
```

Comment:
These elsif chains are long enough that we might want to consider data files for this fixture data (json or yaml or something)